blocks/resources/block.php

```diff
@@ -50,6 +51,17 @@ function render_callback( array $attributes ) : string {
 	$resources_query = new AirpressQuery( 'Resources', 0 );
 	$resources_query->addFilter( '{Publish Status of Resource} = "Published"' );
 
+	$trp           = TRP_Translate_Press::get_trp_instance();
```

Comment:
Let's add a `class_exists` conditional here? This will fatal error if TranslatePress isn't installed, which is true for staging.

Reply:
Good catch